numba/tests/test_np_functions.py

```diff
@@ -2503,6 +2507,43 @@ def check_pass_through(jitted, expect_same, params):
                 else:
                     check_pass_through(cfunc, True, params)
 
+    def test_repeat(self):
+        pyfunc = np_repeat
+        cfunc = jit(nopython=True)(pyfunc)
+
+        def check(a, repeats):
+            np.testing.assert_equal(pyfunc(a, repeats), cfunc(a, repeats))
+
+        target_inputs = [
+            np.ones(1),
+            np.arange(1000),
+            np.array([[0, 1], [2, 3]]),
```

Comment:
Perhaps use a few different dtypes in here, in particular `int32` and complex types.